### ISSUE TITLE
This plugin is not consistent with many of the practices followed by most plugins

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugin#175]
I see several things just looking around that we should change to bring it more inline with the way that all the other plugins are doing various things these are what I found when doing a release:
- changelog does not follow the same format
- the tags are all using the silly `v$VERSION` which I think is silly, I never understood why people liked this it's a number if you can't figure out that its a version number looking at something reasonable then you probably should not be software dev. Anyways enough of my ranting just pointing out areas in where there are differences and we should standardize where possible. 
- version is not broken out into its own file and is not setup the same way

I am sure there are plenty others...

**Comment by portertech** [comment]
These all seem like reasonable suggestions. It is important to remember that this library is not a plugin, but the very backbone of every Sensu community plugin, and its practices may differ due to its very nature.

**Comment by majormoses** [comment]
Agreed that it is different and does not need to be in 100% alignment but none of these things that were brought up seem plugin specific.

**Comment by majormoses** [comment]
only one I see that might be different would be added rake tasks and a possibly modified rubocop config.

**Comment by luisdavim** [comment]
Hi, I've just opened #177 that addresses the rubocop part, the configuration I've added is not exactly the same as the one in the skel plugin but it's a start.
